modules/executives/api/fetchExecutives.ts

```diff
@@ -1,17 +1,19 @@
 import { config } from 'lib/config';
-import { EXEC_PROPOSAL_INDEX } from 'lib/constants';
+import { EXEC_PROPOSAL_INDEX, SupportedNetworks } from 'lib/constants';
 import { fsCacheGet, fsCacheSet } from 'lib/fscache';
 import { fetchGitHubPage } from 'lib/github';
-import { isTestnet } from 'lib/maker';
-import { CMSProposal } from 'types/proposal';
+import { getNetwork, isTestnet } from 'lib/maker';
+import { CMSProposal } from 'modules/executives/types';
 import mockProposals from './mocks/proposals.json';
 import { parseExecutive } from './parseExecutive';
 import invariant from 'tiny-invariant';
 import { markdownToHtml } from 'lib/utils';
 
-export async function getExecutiveProposals(): Promise<CMSProposal[]> {
+export async function getExecutiveProposals(network?: SupportedNetworks): Promise<CMSProposal[]> {
+  const currentNetwork = network ? network : getNetwork();
+
     if (config.USE_FS_CACHE) {
-      const cachedProposals = fsCacheGet('proposals');
+      const cachedProposals = fsCacheGet(`proposals-${currentNetwork}`);
```

Comment:
Will this allow us to use the cache on kovan?

Reply:
yeah, at least we can split into 2 cache files instead of returning the same ones. I still don't know much about how kovan execs are pulled